index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">

```

Comment:
Great job so far! You just need to add some small changes.

Please add on head meta:
-Title
-Keywords.
-Description.
-Favicon.

These changes are going to make the website more SEO friendly.

If you want to know more specifics about how to make the site SEO-friendly, check this:

[https://developers.google.com/search/docs/crawling-indexing/special-tags](https://github.com/MarcoDDM/trials-review/pull/url)